files/mailer.rb

```diff
@@ -54,12 +54,12 @@ def mail_destination
   end
 
   def handle
-    
+
     mail_to = mail_destination
     # Only procede if we have an email address to work with
     return false unless mail_to
 
-    mail_from = 'sensu@yelp.com'
+    mail_from = settings['mailer']['mail_from'] || 'sensu@yelp.com'
```

Comment:
Maybe make it handler_settings['mail_from'] like other settings, for consistency?